packages/block/src/from-rpc.ts

```diff
@@ -44,5 +44,5 @@ export default function blockFromRpc(blockParams: any, uncles: any[] = [], optio
 
   const uncleHeaders = uncles.map((uh) => blockHeaderFromRpc(uh, options))
 
-  return Block.fromBlockData({ header, transactions, uncleHeaders })
+  return Block.fromBlockData({ header, transactions, uncleHeaders }, options)
```

Comment:
This is a bug fix 🥴 I found common wasn't being passed properly to the returned block in `blockFromRpc` when using it for the [goerli block test](https://github.com/ethereumjs/ethereumjs-monorepo/pull/1032/files#diff-5e26767037750597317a815e2bc28ff13e85897dde318118ad5a392d9ba10ddaR105).